### DISCUSSION TITLE
CSS styles supported by html-to-docx

### DISCUSSION THREAD

**Discussion by rgarciape** [privateOmega/html-to-docx#134] [Q&A]
Hi there!

I'm working with the library and was wondering the limitations when including some CSS properties.

Concretely, when using flex CSS properties:
`.title_section {
    display: flex;
    flex-direction: column;
    align-items: center;
    gap: 12px;
}`
Desired style:
![Captura de Pantalla 2022-06-12 a las 11 35 24](https://user-images.githubusercontent.com/11663001/173227095-1a91049a-b3d2-4062-a603-e7237ddd1734.png)

Result:
![Captura de Pantalla 2022-06-12 a las 11 36 25](https://user-images.githubusercontent.com/11663001/173227118-3f81226d-2570-4f92-acb6-537b001da8b0.png)

Which CSS properties can and cannot be used.

Thanks a lot in advance and congrats to  developers and contributors

**Comment by UrbaneD0ge** [comment]
Hi! Hoping for more clarity on what's supported as I'm working with this package in a very basic node/express app.

I'm trying to emulate an existing word doc format used in my job, but I don't seem to be able to implement the styles that I'm expecting.

Example: Word describes the table borders I want as "1/4pt" - but applying borders as ".25pt" in the inline style attribute results in what word seems to classify as '1/2pt'. Also trying to bring in a fill color and don't know where to start on that.

Thanks!

**Comment by privateOmega** [comment]
@rgarciape @UrbaneD0ge Not all styles are supported yet and I dont have a list of what styles work and what doesnt since I was mostly focussed on basic structure of it and using basic styles like bold, italics, underline, etc. One thing I know for sure is, there would never be a day when this package converts html to a docx file with full fidelity(its never possible).

@UrbaneD0ge Table border isnt properly developed and might not be perfect, feel free to raise a PR to fix it, I rely on opensource contributions as well, since I dont have time to commit nowadays.
